packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -14,16 +14,17 @@ import { store as editSiteStore } from '../../store';
 export default function useInitEditedEntityFromURL() {
 	const { params: { postId, postType } = {} } = useLocation();
 	const { isRequestingSite, homepageId, url } = useSelect( ( select ) => {
-		const { getSite } = select( coreDataStore );
+		const { getSite, getUnstableBase } = select( coreDataStore );
 		const siteData = getSite();
+		const base = getUnstableBase();
 
 		return {
-			isRequestingSite: ! siteData,
+			isRequestingSite: ! base,
 			homepageId:
 				siteData?.show_on_front === 'page'
 					? siteData.page_on_front
 					: null,
-			url: siteData?.url,
```

Comment:
Code-wise, this is in line with the other two places I've seen this in use: [home-link](https://github.com/WordPress/gutenberg/blob/28c371a878e5353f2d47bd0707bce95abca130e1/packages/block-library/src/home-link/edit.js#L27) and [header-edit-mode](https://github.com/WordPress/gutenberg/blob/28c371a878e5353f2d47bd0707bce95abca130e1/packages/edit-site/src/components/header-edit-mode/index.js#L86).

Reply:
This seems the proper way to query this data based on usage, but I'm curious about the story of the unstableBase. I see it defined as [an entity](https://github.com/WordPress/gutenberg/blob/28c371a878e5353f2d47bd0707bce95abca130e1/packages/core-data/src/entities.js#L31) and the data comes from `/` in the network tab. However, I don't find this endpoint documented [here](https://developer.wordpress.org/rest-api/reference/).

Reply:
@oandregal, the index endpoint data is defined here - https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/class-wp-rest-server.php#L1255-L1266.

The change correctly uses `home` instead of the `url`.